jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/BeanPropertyIntrospector.java

```diff
@@ -1,189 +1,149 @@
 package com.fasterxml.jackson.jr.ob.impl;
 
+import com.fasterxml.jackson.jr.ob.impl.POJODefinition.Prop;
+import com.fasterxml.jackson.jr.ob.impl.POJODefinition.PropBuilder;
+
 import java.lang.reflect.Constructor;
 import java.lang.reflect.Field;
-import java.lang.reflect.Method;
 import java.lang.reflect.Modifier;
+import java.util.Arrays;
```

Comment:
As per Jackson code style import order for non-statics should be:

1. JDK imports
2. 3rd party imports, if any
3. Core jackson imports if any
4. Imports from package itself